cli/command/context/testdata/list.golden

```diff
@@ -1,5 +1,5 @@
-NAME        DESCRIPTION                               DOCKER ENDPOINT
-current *   description of current                    https://someswarmserver.example.com
-default     Current DOCKER_HOST based configuration   unix:///var/run/docker.sock
-other       description of other                      https://someswarmserver.example.com
-unset       description of unset                      https://someswarmserver.example.com
+NAME        DESCRIPTION                               DOCKER ENDPOINT                       ERROR
```

Comment:
`ERROR` looks like a copy/paste mistake here

Reply:
It's the title for the `ERROR` column that was added, but looking at this, it looks like I have more work to do. The context store itself already fails "full stop" if it encounters an error, so without some of the other changes from https://github.com/docker/cli/pull/3847, it looks like this doesn't work 😞